### ISSUE TITLE
crash when starting the game

### ISSUE THREAD

**Issue by MarxMustermann** [MarxMustermann/OfMiceAndMechs#23]
from discord

> When trying to run the game, i got errors due to lack of 'fcntl' in urwid.
> [10:43]
> I'm pretty sure this is because of this line:
> [10:43]
> try:
>     import fcntl
>     import termios
>     import tty
> except ImportError:
>     pass # windows
> [10:43]
> Do you  know how to avoid this error?

more information needed to reproduce hunt bug. Offending code is not in the game, probably a dependency issue

**Comment by UmbralReaper** [comment]
Rather than a crash when starting the game, the game did not even start. Upon opening the ExecuteMe.py file I got an error message that basically boiled down into: lack of 'fcntl' in urwid. 

When I read the code the stack back trace directed me to, I saw 
```
try:
import fcntl
import termios
import tty
except ImportError:
pass # windows
```
Due to the combination of ImportError and # windows, I assumed that it was because, in Windows, urwid skips the import and thus causes errors.
